### PR TITLE
SEQ-7243: Read standard AGE output.

### PR DESCRIPTION
**Pull request by sbandara**
These changes to `age_parser.py` allow us to read AGE outputs in the format used by the main developer's branch, so MetaSV will not be decoupled from future improvements to AGE. Correspondingly, I removed the instructions to download AGE from the `simple-parseable-output` branch.

@marghoob, @johnmu, and Mohammad, I placed comments labeled `ATTN` in a few different places where AGE output even by the `simple-parseable-output` branch is more detailed than what MetaSV uses, or similarly, where both AGE branches provide information on multiple segments or boundaries although MetaSV only inspects the first of each. So I thought those sections may be especially critical for you to look at when reviewing this PR.

### CODE REVIEW

**Comment by sbandara** [comment]
What follows is a simple test stub that outputs the parser results for a given AGE file. Maybe you don't want to merge this one, but it will certainly help validating the parser.

**Comment by johnmu** [comment]
@msahraeian can you test this to make sure it does this right thing?

File: metasv/age_parser.py [hunk 1]
```diff
@@ -73,39 +81,101 @@ def __init__(self, age_out_file=None,tr_region_1=[]):
...
+            start_end.append([int(m.group(1)), int(m.group(2))])
+        return start_end
+
+    def parse_input_descriptor(self, line):
+        """Returns a tuple of filename and sequence length."""
+        m = self.rx_input.search(line)
+        if m is None:
+            raise AgeFormatError("INPUT DESCRIPTOR", age_fd.line_num)
+        return (m.group(2), int(m.group(1)))
+
```
Comment (msahraeian) [inline metasv/age_parser.py]: I started testing this and got an error because "age_fd is not defined in parse_input_descriptor"

**Comment by sbandara** [comment]
@msahraeian, thanks for testing! The fact that parsing the input descriptor fails is interesting independent of the fact that the error object could not be constructed (which I just fixed). Could you send me the AGE file that made you observe this?

**Comment by sbandara** [comment]
I validated this fix against runs by @msahraeian for chromosome 22 of NA12878. The result VCF is identical between (a) bioinform/metasv/master + marghoob/age/simple-parseable-output and (b) this fix + abyzovlab/age/master. To run this on Mac OS X, I had to merge this branch into a separate branch for SEQ-7207 in sbandara/metasv (no PR yet). The result files have `diff`ering lines, but all of those are only because of unordered method entries in the `SOURCE` field. This means, I have not tested on Linux yet.

@msahraeian, could you validate by checking out **this** SEQ-7243 branch + abyzov/age/master on **Linux** with at least one different chromosome, e.g. 21? The result would have to be identical to running bioinform/metasv/master + marghoob/age/simple-parseable-output. @marghoob may be able to merge this PR then if it passes.

**Comment by marghoob** [comment]
Thanks @sbandara for fixing this long-standing issue.

**Comment by msahraeian** [comment]
Results seems identical on Linux too.

**Comment by sbandara** [comment]
Thanks @msahraeian for testing and closing SEQ-7243. Related news is that Dr. Abyzov merged my PR for the `-allpos` flag which you guys may be interested in for future use in MetaSV (for always showing identities). @marghoob I saw you output these in your `simple-parseable-output` branch but not use them yet.

**Pull request merged**